src/towncrier/test/test_check.py

```diff
@@ -204,6 +213,44 @@ def test_fragment_exists_but_not_in_check(self):
                 (result.output, str(fragment_path)),
             )
 
+    def test_fragment_exists_and_staged(self):
+        """A fragment that exists but is marked as check=False is ignored by the check."""
```

Comment:
Please update this [test docstring](https://jml.io/test-docstrings/) to mention staging.

Reply:
fixed in 363532aa92fc31c5d46c6bea2cd4760a7604c4f9